samples/http-client-examples/server.ts

```diff
@@ -1,41 +1,41 @@
 import express from 'express';
 const app = express();
 
-app.get('', async (req, res) => {
+app.get('', async (req: express.Request, res: express.Response) => {
```

Comment:
```suggestion
const { Request, Response } = express;

app.get('', async (req: Request, res: Response) => {
```
[pp]

Reply:
I've configured auto-merge so I could not address your comment, I don't have a strong opinion on that. Honestly I was not aware of this syntax.

Reply:
Wanted to say +1 on florians comment, but I would even prefer this:
`import { Request, Response } from "express";`